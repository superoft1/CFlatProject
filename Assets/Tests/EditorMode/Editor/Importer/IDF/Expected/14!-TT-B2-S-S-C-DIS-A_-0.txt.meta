fileFormatVersion: 2
guid: 26db86aacfdf5ef40a00540a1fb31f9b
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
