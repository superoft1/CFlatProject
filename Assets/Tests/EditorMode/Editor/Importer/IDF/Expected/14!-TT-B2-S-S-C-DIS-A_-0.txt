WeldNeckFlange, , (0.0, 0.0, -1.0), (-124.0, 123.0, 1.5), False
(-124.0, 123.0, 1.5)
(-124.0, 123.0, 1.6)
FreeVertex: 1
Pipe, , (0.0, 0.0, 1.0), (-124.0, 123.0, 2.3), False
(-124.0, 123.0, 1.6)
(-124.0, 123.0, 3.0)
FreeVertex: 0
ConcentricPipingReducer, , (0.0, 0.0, 1.0), (-124.0, 123.0, 3.2), False
(-124.0, 123.0, 3.4)
(-124.0, 123.0, 3.0)
FreeVertex: 0
Elbow90, , (1.0, 0.0, 0.0), (-124.0, 123.0, 3.9), False
(-124.0, 123.0, 3.4)
(-124.5, 123.0, 3.9)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-124.6, 123.0, 3.9), False
(-124.5, 123.0, 3.9)
(-124.6, 123.0, 3.9)
FreeVertex: 0
Elbow90, , (0.0, -1.0, 0.0), (-125.2, 123.0, 3.9), False
(-124.6, 123.0, 3.9)
(-125.2, 123.5, 3.9)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-125.2, 123.7, 3.9), False
(-125.2, 123.5, 3.9)
(-125.2, 124.0, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, 1.0, 0.0), (-125.2, 124.0, 3.9), False
(-125.2, 124.1, 3.9)
(-125.2, 124.0, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-125.2, 124.2, 3.9), False
(-125.2, 124.1, 3.9)
(-125.2, 124.2, 3.9)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-125.2, 124.7, 3.9), False
(-125.2, 124.2, 3.9)
(-125.2, 125.2, 3.9)
FreeVertex: 0
WeldOlet, , (-1.0, 0.0, 0.0), (-125.2, 125.2, 3.9), False
(-125.2, 125.2, 3.9)
(-125.2, 125.2, 3.9)
(-125.4, 125.2, 3.9)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-125.4, 125.2, 3.9), False
(-125.4, 125.2, 3.9)
(-125.5, 125.2, 3.9)
FreeVertex: 0
GlobeValve, , (-1.0, 0.0, 0.0), (-125.5, 125.2, 3.9), False
(-125.5, 125.2, 3.9)
(-125.5, 125.2, 3.9)
FreeVertex: 0
PressureGauge, PG_102_11_10_1070_1E, (-1.0, 0.0, 0.0), (-125.6, 125.2, 3.9), False
(-125.6, 125.2, 3.9)
(-125.5, 125.2, 3.9)
FreeVertex: 1
Pipe, , (0.0, 1.0, 0.0), (-125.2, 125.4, 3.9), False
(-125.2, 125.2, 3.9)
(-125.2, 125.6, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, 1.0, 0.0), (-125.2, 125.7, 3.9), False
(-125.2, 125.7, 3.9)
(-125.2, 125.6, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-125.2, 125.8, 3.9), False
(-125.2, 125.7, 3.9)
(-125.2, 125.8, 3.9)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-125.2, 126.2, 3.9), False
(-125.2, 125.8, 3.9)
(-125.2, 126.6, 3.9)
FreeVertex: 0
Tee, , (0.0, 1.0, 0.0), (-125.2, 126.9, 3.9), True
(-125.2, 127.2, 3.9)
(-125.2, 126.6, 3.9)
(-124.9, 126.9, 3.9)
FreeVertex: 0
Pipe, , (1.0, 0.0, 0.0), (-124.8, 126.9, 3.9), False
(-124.9, 126.9, 3.9)
(-124.7, 126.9, 3.9)
FreeVertex: 0
Elbow90, , (0.0, 1.0, 0.0), (-124.5, 126.9, 3.9), False
(-124.7, 126.9, 3.9)
(-124.5, 126.7, 3.9)
FreeVertex: 0
Pipe, , (0.0, -1.0, 0.0), (-124.5, 126.6, 3.9), False
(-124.5, 126.7, 3.9)
(-124.5, 126.6, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-124.5, 126.6, 3.9), False
(-124.5, 126.5, 3.9)
(-124.5, 126.6, 3.9)
FreeVertex: 1
WeldNeckFlange, , (0.0, 1.0, 0.0), (-125.2, 127.2, 3.9), False
(-125.2, 127.3, 3.9)
(-125.2, 127.2, 3.9)
FreeVertex: 0
CheckValve, , (0.0, 1.0, 0.0), (-125.2, 127.4, 3.9), True
(-125.2, 127.5, 3.9)
(-125.2, 127.3, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-125.2, 127.6, 3.9), False
(-125.2, 127.5, 3.9)
(-125.2, 127.6, 3.9)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-125.2, 127.7, 3.9), False
(-125.2, 127.6, 3.9)
(-125.2, 127.8, 3.9)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-125.2, 127.8, 3.9), False
(-125.2, 127.8, 3.9)
(-125.2, 127.8, 3.9)
(-125.2, 127.8, 3.7)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-125.2, 127.8, 3.7), False
(-125.2, 127.8, 3.7)
(-125.2, 127.8, 3.6)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-125.2, 127.8, 3.6), False
(-125.2, 127.8, 3.5)
(-125.2, 127.8, 3.6)
FreeVertex: 1
Pipe, , (0.0, 1.0, 0.0), (-125.2, 127.9, 3.9), False
(-125.2, 127.8, 3.9)
(-125.2, 128.0, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, 1.0, 0.0), (-125.2, 128.0, 3.9), False
(-125.2, 128.1, 3.9)
(-125.2, 128.0, 3.9)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-125.2, 128.1, 3.9), False
(-125.2, 128.1, 3.9)
(-125.2, 128.1, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-125.2, 128.2, 3.9), False
(-125.2, 128.1, 3.9)
(-125.2, 128.3, 3.9)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-125.2, 128.3, 3.9), False
(-125.2, 128.3, 3.9)
(-125.2, 128.4, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, 1.0, 0.0), (-125.2, 128.5, 3.9), False
(-125.2, 128.5, 3.9)
(-125.2, 128.4, 3.9)
FreeVertex: 0
GateValve, , (0.0, 1.0, 0.0), (-125.2, 128.7, 3.9), False
(-125.2, 128.9, 3.9)
(-125.2, 128.5, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-125.2, 129.0, 3.9), False
(-125.2, 128.9, 3.9)
(-125.2, 129.0, 3.9)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-125.2, 129.1, 3.9), False
(-125.2, 129.0, 3.9)
(-125.2, 129.2, 3.9)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-125.2, 129.2, 3.9), False
(-125.2, 129.2, 3.9)
(-125.2, 129.2, 3.9)
(-125.2, 129.2, 3.7)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-125.2, 129.2, 3.7), False
(-125.2, 129.2, 3.7)
(-125.2, 129.2, 3.6)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-125.2, 129.2, 3.6), False
(-125.2, 129.2, 3.5)
(-125.2, 129.2, 3.6)
FreeVertex: 1
Pipe, , (0.0, 1.0, 0.0), (-125.2, 129.3, 3.9), False
(-125.2, 129.2, 3.9)
(-125.2, 129.4, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, 1.0, 0.0), (-125.2, 129.5, 3.9), False
(-125.2, 129.5, 3.9)
(-125.2, 129.4, 3.9)
FreeVertex: 0
GateValve, , (0.0, 1.0, 0.0), (-125.2, 129.7, 3.9), False
(-125.2, 129.9, 3.9)
(-125.2, 129.5, 3.9)
FreeVertex: 0
WeldNeckFlange, , (0.0, -1.0, 0.0), (-125.2, 130.0, 3.9), False
(-125.2, 129.9, 3.9)
(-125.2, 130.0, 3.9)
FreeVertex: 0
Pipe, , (0.0, 1.0, 0.0), (-125.2, 130.1, 3.9), False
(-125.2, 130.0, 3.9)
(-125.2, 130.2, 3.9)
FreeVertex: 0
WeldOlet, , (0.0, 0.0, -1.0), (-125.2, 130.2, 3.9), False
(-125.2, 130.2, 3.9)
(-125.2, 130.2, 3.9)
(-125.2, 130.2, 3.7)
FreeVertex: 0
Pipe, , (0.0, 0.0, -1.0), (-125.2, 130.2, 3.7), False
(-125.2, 130.2, 3.7)
(-125.2, 130.2, 3.6)
FreeVertex: 0
GateValve, , (0.0, 0.0, -1.0), (-125.2, 130.2, 3.6), False
(-125.2, 130.2, 3.5)
(-125.2, 130.2, 3.6)
FreeVertex: 1
Pipe, , (0.0, 1.0, 0.0), (-125.2, 130.6, 3.9), False
(-125.2, 130.2, 3.9)
(-125.2, 130.9, 3.9)
FreeVertex: 0
Elbow90, , (0.0, 0.0, -1.0), (-125.2, 131.5, 3.9), False
(-125.2, 130.9, 3.9)
(-125.2, 131.5, 4.4)
FreeVertex: 0
Pipe, , (0.0, 0.0, 1.0), (-125.2, 131.5, 5.0), False
(-125.2, 131.5, 4.4)
(-125.2, 131.5, 5.5)
FreeVertex: 0
Elbow90, , (1.0, 0.0, 0.0), (-125.2, 131.5, 6.0), False
(-125.2, 131.5, 5.5)
(-125.7, 131.5, 6.0)
FreeVertex: 0
Pipe, , (-1.0, 0.0, 0.0), (-126.7, 131.5, 6.0), False
(-125.7, 131.5, 6.0)
(-127.6, 131.5, 6.0)
FreeVertex: 0
Tee, , (-1.0, 0.0, 0.0), (-127.9, 131.5, 6.0), True
(-128.2, 131.5, 6.0)
(-127.6, 131.5, 6.0)
(-127.9, 131.5, 6.3)
FreeVertex: 2
FreeVertexTotal: 8
